packages/node-cli/src/CLI.js

```diff
@@ -49,18 +54,49 @@ module.exports = class CLI {
 
 			program
 				.version(CLI_VERSION, '--version')
-				.option(`${INTERACTIVE_ALIAS}, ${INTERACTIVE_OPTION}`, NodeTranslationService.getMessage(INTERACTIVE_OPTION_DESCRIPTION))
-				.on('command:*', args => {
+				.option(
+					`${INTERACTIVE_ALIAS}, ${INTERACTIVE_OPTION}`,
+					NodeTranslationService.getMessage(INTERACTIVE_OPTION_DESCRIPTION),
+					this._validateInteractive
```

Comment:
is this the right place to add this.validateinteractive?  I see from the definition that this parameter is for the default value ?
option(flags: string, description?: string, defaultValue?: string | boolean):

Reply:
validateInteractive is a function so the definition that applies is the following one:

`option<T>(flags: string, description: string, fn: (value: string, previous: T) => T, defaultValue?: T): this;`

I have search the example in the documentation of [commander](https://github.com/tj/commander.js#custom-option-processing)

> You may specify a function to do custom processing of option-arguments. The callback function receives two parameters, the user specified option-argument and the previous value for the option. It returns the new value for the option.
> 
> This allows you to coerce the option-argument to the desired type, or accumulate values, or do entirely custom processing.
> 
> You can optionally specify the default/starting value for the option after the function parameter.

In this case, I am not passing any value to the function because I don't need to, because the variable that I'm using is already in the global scope of the class.

Reply:
got it. I forgot that the defaultvalue parameter is optional. looks good. thanks!